pkg/kn/commands/service/describe_test.go

```diff
@@ -495,7 +496,7 @@ func validateServiceOutput(t *testing.T, service string, output string) {
 	assert.Assert(t, util.ContainsAll(output, "Ready", "RoutesReady", "OK", "TYPE", "AGE", "REASON"))
 }
 
-func createTestService(name string, revisionNames []string, conditions duckv1.Conditions) v1alpha1.Service {
+func createTestService(name string, revisionNames []string, serviceaccount string, conditions duckv1.Conditions) v1alpha1.Service {
```

Comment:
Can we define `createTestServiceWithServiceAccount` function and have it called `createTestService` + code for adding ServiceAccount ?  
Most of the invocation to createTestService are with empty strings otherwise.

Reply:
*serviceAccount

Reply:
Fixed. Thanks.